frontend/src/app/todo/list/spinner.service.ts

```diff
@@ -7,16 +7,16 @@ import { BehaviorSubject, Observable } from 'rxjs';
 })
 export class SpinnerService
```

Comment:
LoaderStore ...

This is not an official Angular naming convention but I use .service for http only operations and .store for state handling